### ISSUE TITLE
Hamburger Menu Colors - Unable to see selected item

### ISSUE THREAD

**Issue by darenm** [Windows-XAML/Template10#200]
![hamburgercolorswrong](https://cloud.githubusercontent.com/assets/1674843/10406095/6643b3f6-6e92-11e5-9c54-fb959c10c2ab.gif)

**Comment by dg2k** [comment]
The HamburgerMenu and PageHeader have a few issues I have noticed. PR #195 will, for the most part, address these issues (but lately noticed further outstanding style corrections to be made). There are a number of things confusing the issue here: 1) when Custom.xaml was split and then recombined, I think, somethings got changed/lost on the way (sorry if this sounds a blame, it isn't); 2) the AccentColor which I believe is meant for testing?? does some overriding/changing of color schemes so Custom.xaml definition may not appear as it should.

I know PR #195 will take us a step closer to addressing the style issues and then Custom.xaml should work as expected.

I checked if modifying Custom.xaml alone fixes the problem but didn't work without the fixes in PR #195 (also noting that AccentColor has some impact).
